client/apps/edit/components/header/test/index.coffee

```diff
@@ -30,7 +30,7 @@ describe 'EditHeader', ->
 
     it 'indicates saving on change', ->
       @view.article.trigger 'change'
-      @view.$('#edit-save').hasClass('is-saving').should.be.ok
```

Comment:
This is incorrect and doesn't fail whether the class is there or not. The check needs to be `equal true/false`.